drivers/include/periph/rtc.h

```diff
@@ -58,6 +63,55 @@ extern "C" {
 #define RIOT_EPOCH (2020)
 #endif
 
+#if defined(MODULE_SYS_BUS_RTC) || DOXYGEN
+/**
+ * @brief INTERNAL USE ONLY: Post a time change notification.
+ *
+ * For use in time setter functions of RTC and RTC_RTT related
+ * implementations.
+ *
+ * @param[in] s_before      The second part of the time stamp
+ *                          before the update
+ * @param[in] us_before     The microsecond part of the time stamp
+ *                          before the update
+ * @param[in] s_after       The second part of the time stamp
+ *                          after the update
+ * @param[in] us_after      The microsecond part of the time stamp
+ *                          after the update
+ * @param[in] do_post       A switch for turning off messages to
+ *                          the bus, intended for testing
+ *
+ * @return                  The calculated time difference,
+ *                          intended for testing
+ */
+static inline int32_t sys_bus_rtc_util_post_timechange(
```

Comment:
does this have to be inline?

Reply:
I'll try to move it to `drivers/periph_common/rtc.c` which should be translated under all conditions.

Reply:
I moved the definition to the compilation unit as written above and also changed the somewhat ugly name of the function.